src/js/api/posts/create.mjs

```diff
@@ -0,0 +1,26 @@
+import { authToken } from "../tokenFetch.mjs";
+
+/**
+ *
+ * @param {string} postUrl api url
+ * @param {string} method method used in api call
+ * @param {string} post data that is retrieved
+ * @returns {string} returns api data based on input (url, method, data)
+ */
+
+async function postData(postUrl, method, post) {
+  try {
+    console.log("postUrl", postUrl);
+    const response = await authToken(postUrl, {
```

Comment:
What happens if `response.ok === false`? The response could be a 404, that's a server error not a javaascript error and won't trigger the catch block.

Reply:
I added more error checks, it should be better now